### PR TITLE
Add support for specifying full image url as alternative to image alias.

### PR DESCRIPTION
**Pull request by duftler**
Fix incorrect field name in test.

@jadair-cloudera please review.

### CODE REVIEW

**Comment by duftler** [comment]
screen shot 2015-08-21 at 8 20 32 pm
screen shot 2015-08-21 at 8 21 07 pm
screen shot 2015-08-21 at 8 21 18 pm
screen shot 2015-08-21 at 8 21 35 pm

**Comment by duftler** [comment]
Closes #97.

File: provider/src/main/java/com/cloudera/director/google/compute/GoogleComputeInstanceTemplateConfigurationProperty.java [hunk 1]
```diff
@@ -29,10 +29,10 @@
 
   IMAGE(new SimpleConfigurationPropertyBuilder()
       .configKey(ComputeInstanceTemplateConfigurationPropertyToken.IMAGE.unwrap().getConfigKey())
-      .name("Image Alias")
+      .name("Image Alias or URL")
       .addValidValues("centos6", "rhel6")
-      .defaultDescription("The image alias from plugin configuration.")
-      .defaultErrorMessage("Image alias is mandatory")
+      .defaultDescription("The image alias from plugin configuration or a full image url.")
```
Comment (jadair-cloudera) [inline provider/src/main/java/com/cloudera/director/google/compute/GoogleComputeInstanceTemplateConfigurationProperty.java]: Capitalize URL in error messages here and below.

**Comment by duftler** [comment]
Done.

**Pull request merged**